CMakeLists.txt

```diff
@@ -98,7 +98,6 @@ if(CAF_USE_CCACHE)
   if(CCACHE_PROGRAM)
     message(STATUS "Using ccache command: ${CCACHE_PROGRAM}")
     set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK "${CCACHE_PROGRAM}")
```

Comment:
Seems like an orthogonal concern to the purpose of this commit/PR.

Reply:
Yeah, that made it in out of accident, I just had it fixed for my personal use. I can revert if you want to.

Reply:
I'm fine with the change, as long as fixes an issue for you. I merely would've liked to see this in its own commit with a brief explanation what it solves. :)

Reply:
Not a real issue, but [ccache doesn't support linking and just passes the command back to the regular compiler driver](https://ccache.samba.org/manual/latest.html#_limitations). Enabling it for the linker is pointless.

Reply:
👍